crates/bevy_window/src/window.rs

```diff
@@ -839,7 +859,7 @@ impl Default for WindowDescriptor {
             position: WindowPosition::Automatic,
             resize_constraints: WindowResizeConstraints::default(),
             scale_factor_override: None,
-            present_mode: PresentMode::Fifo,
+            present_mode: PresentMode::AutoVsync,
```

Comment:
Stick with Fifo. Universally supported, possibly all anyone should ever need with frame pacing, known behaviour. :)